pkg/ccl/backupccl/restore_old_versions_test.go

```diff
@@ -346,7 +346,7 @@ func fullClusterRestoreSystemRoleMembersWithoutIDs(exportDir string) func(t *tes
 		// manifest version is always less than the MinSupportedVersion which will
 		// in turn fail the restore unless we pass in the specified option to elide
 		// the compatibility check.
-		sqlDB.Exec(t, fmt.Sprintf("RESTORE FROM '%s' WITH UNSAFE_RESTORE_INCOMPATIBLE_VERSION", localFoo))
+		sqlDB.Exec(t, fmt.Sprintf("RESTORE FROM '/' IN '%s' WITH UNSAFE_RESTORE_INCOMPATIBLE_VERSION", localFoo))
```

Comment:
The testdata backup data was not backed up using the new syntax, and thus missing the `metadata` folder. I'm not quite sure how the data was created, but this works as a quick and easy fix.